src/store/metadata/stub/content/editions/aged-19-years-and-over-further-education-and-skills-learner-achievements_2023-05.json

```diff
@@ -0,0 +1,156 @@
+        "keywords": [
+          "skills",
+          "education",
+          "adults",
+          "subnational",
+          "county",
+          "region"
+        ],
+        "licence": "http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+        "spatial_resolution":["E06", "E07", "E08", "E09", "E11"],
+        "spatial_coverage": "E92000001",
+        "temporal_resolution": ["P1Y"],
+        "temporal_coverage": {
+          "start": "2016-08-01T00:00:00Z",
+          "end": "2021-08-01T00:00:00Z"
+        },
+        "versions_url": "https://staging.idpd.uk/datasets/aged-19-years-and-over-further-education-and-skills-learner-achievements/editions/2023-05/versions",
+        "versions": [
+          {
+            "@id": "https://staging.idpd.uk/datasets/aged-19-years-and-over-further-education-and-skills-learner-achievements/editions/2023-05/versions/1",
+            "issued": "2023-05-17T12:41:00Z"
+          }
+        ],
+        "table_schema": {
+          "columns": [
+            {
+                "name": "period",
+                "datatype": "string",
+                "titles": "gregorian-interval",
+                "description": "The time period associated with the observation."
+              },
+              {
+                "name": "areacd",
+                "datatype": "string",
+                "titles": "Statistical Geography",
+                "description": "The geography code associated with the observation."
+              },
+              {
+                "name": "areanm",
+                "datatype": "string",
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
+                "name": "variable_name",
+                "datatype": "string",
+                "titles": "Variable Name",
+                "description": "Which ESS variable is being measured"
+              },
+              {
+                "name": "indicator", 		
+                "datatype": "string",
+                "titles": "Indicator",
+                "description": "Which ESS indicator is being measured"
+              },
+              {
+                "name": "mission",
+                "datatype": "string",
+                "titles": "Mission",
+                "description": "Which ESS mission number this dataset is fulfilling"
+              },
+              {
+                "name": "category",
+                "datatype": "string",
+                "titles": "Category",
+                "description": "Which ESS category the dataset belongs to"
+              },
+              {
+                  "name": "observation",
+                  "datatype": "decimal",
```

Comment:
I think the datatype should be changed from "decimal" to `"integer"` as this dataset is about the count of learners achieving a funded further education and skills learning